package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@natlibfi/melinda-rest-api-importer",
+  "description": "API microservice of Melinda record batch import system",
+  "author": {
+    "name": "The National Library of Finland"
+  },
+  "keywords": [],
+  "homepage": "https://github.com/NatLibFi/melinda-rest-api-importer",
+  "bugs": {
+    "url": "https://github.com/NatLibFi/melinda-rest-api-importer/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:NatLibFi/melinda-rest-api-importer.git"
+  },
+  "license": "AGPL-3.0+",
+  "version": "0.0.1",
+  "main": "./dist/index.js",
+  "engines": {
+    "node": "10"
+  },
+  "private": true,
+  "scripts": {
```

Comment:
Make this similar to other repositories' package.json script (At least test & test:dev must be present)